infra/eligibility_screener/template/ecs.tf

```diff
@@ -11,6 +15,14 @@ resource "aws_security_group" "allow-screener-traffic" {
     protocol    = "tcp"
     cidr_blocks = ["0.0.0.0/8"]
   }
+  ingress {
+    description = "VPC CIDR; allows healthchecks"
+    from_port   = 8080
+    protocol    = "tcp"
+    to_port     = 8080
```

Comment:
where does 8080 get used? 🤔 seems like the load balancer routes LB requests from port 80 to the container on port 3000, so the only necessary ingress rule is the port 3000 rule.